src/pages/Experiment.less

```diff
@@ -38,20 +27,27 @@
         color: @gray;
     }
 
-    &__progress {
-        margin-bottom: 20px;
-    }
-
     &__content {
-        flex-grow: 1;
-        display: flex;
-        flex-direction: column;
+        height: 100%;
     }
 
     &__diff {
-        flex-grow: 1;
-        display: flex;
-        margin-bottom: 20px;
+        height: 100%;
+
+        // must be sum of everything on top + footer
+        // page-header: 50px
+        // experiment-header: 23px
+        // margins: 20px * 2 + 10px
+        // footer: 46+10px
+        // borders: 2px
+        // 50 + 23 + 40 + 56 + 10 + 2 = 181px
+        padding-bottom: 181px;
```

Comment:
What if all that things are variables, so nothing would be broken if some height changes?

Reply:
there is no fixed height for blocks around.
For example height of footer is actually is the height of buttons. Even if you define the height for the footer as soon as you make buttons bigger (or add margin to them or whatever) it will get broken.